### PR TITLE
Add '.bazel' to make-version.js's python build file path

### PR DESCRIPTION
**Pull request by mattsoulanille**
Fix make-version.js to work with the changes made in #7253.

Tested locally with `yarn release-tfjs --git-protocol --dry --use-local-changes --force` since the release tests only run in nightly.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7263)

### CODE REVIEW

File: scripts/make-version.js [hunk 1]
```diff
@@ -61,17 +61,17 @@ version = '${version}'
             `Version file for pip version ${version} saved sucessfully.`);
       });
 
-  const buildFilename = dirName + '/python/BUILD';
+  const buildFilename = dirName + '/python/BUILD.bazel';
```
Comment (chunnienc) [inline scripts/make-version.js:64]: nit: os.path.join ?

Comment (mattsoulanille) [reply scripts/make-version.js]: Good idea. Fixed all instances in the script.

Comment (chunnienc) [reply scripts/make-version.js]: my bad - I thought it's a python file. Thanks for the change!

**Review by Linchenn — APPROVED** [review]
Thank you Matt!

**Review by chunnienc — APPROVED** [review]

**Pull request merged**